pre-processor-scripts/populate-search-fields.js

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable */
+/* Since mongo does not support ES6 we can not lint with our current eslint config */
```

Comment:
You are right Mongo does support ES6. Acknowledged in the description of PR of an earlier PR but this comment mistakenly left in.

I've made the script pass `lint`ing.